cmd/kraft/pkg/pull/pull.go

```diff
@@ -82,6 +83,12 @@ func (opts *Pull) Pre(cmd *cobra.Command, _ []string) error {
 
 	cmd.SetContext(ctx)
 
+	if strings.ContainsRune(opts.Platform, '/') && opts.Architecture == "" {
+		split := strings.SplitN(opts.Platform, "/", 2)
```

Comment:
Also check if the split resulted into 2 strings, otherwise `qemu/` will result in a crash